dll/win32/browseui/shellbrowser.cpp

```diff
@@ -3676,6 +3675,17 @@ LRESULT CShellBrowser::OnToggleStatusBarVisible(WORD wNotifyCode, WORD wID, HWND
     return 0;
 }
 
+LRESULT CShellBrowser::CreateStatusBar()
+{
+    if (!fStatusBar)
+    {
+        fStatusBar = CreateWindow(STATUSCLASSNAMEW, NULL, WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS |
```

Comment:
You should be able to use here `::CreateWindowExW(0, STATUSCLASSNAMEW, NULL, etc.`.